### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by DamienCassou** [purcell/setup-emacs#25]
Thank you very much for this very useful repository!

**Comment by akirak** [comment]
I agree. Even though I am not using setup-emacs myself, but I am extensively using its underlying nix-emacs-ci in my repositories. I started a small support for him on Patreon :)

**Comment by purcell** [comment]
Thanks folks! :heart:
